t/api/02-assets.t

```diff
@@ -84,14 +84,14 @@ for my $i ($iso1, $iso2) {
 
 my $listing = [
     {
-        id       => 6,
+        id       => 7,
```

Comment:
Because I added one to the fixtures, the next id has to be higher. Fixtures are crap :(